oneview_redfish_toolkit/api/zone.py

```diff
@@ -72,10 +80,13 @@ def __init__(self, profile_template, available_targets_obj):
 
         self._validate()
 
-    def fill_resource_blocks(self, profile_template, available_targets):
-        for item in available_targets:
+    def fill_resource_blocks(self):
+        for item in self.available_targets:
```

Comment:
Could this method receives "available_target" as argument? Not depending the instance variable has been set before.

Reply:
Yes, I dont see problem with instance variables in this case, but I will do that.